neqo-transport/src/recovery.rs

```diff
@@ -209,12 +240,173 @@ impl LossRecoverySpaces {
     }
 }
 
+#[derive(Debug)]
+struct CongestionControl {
+    congestion_window: u64, // = kInitialWindow
+    bytes_in_flight: u64,
+    congestion_recovery_start_time: Option<Instant>,
+    ssthresh: u64,
+}
+
+impl Default for CongestionControl {
+    fn default() -> Self {
+        CongestionControl {
+            congestion_window: INITIAL_WINDOW,
+            bytes_in_flight: 0,
+            congestion_recovery_start_time: None,
+            ssthresh: u64::max_value(),
```

Comment:
We apparently don't have u64::max_value() available everywhere.  See https://github.com/mozilla/neqo/pull/356/files#diff-7b309cf4b26d850afe5fe18964666edcR203-R206

Reply:
ok, using `std::u64::MAX` instead